astropy/io/misc/tests/test_pandas.py

```diff
@@ -0,0 +1,90 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from io import StringIO
+
+import pytest
+import numpy as np
+
+from astropy.io import ascii
+from astropy.table import Table, QTable
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.io.misc.pandas import connect
+
+# Check dependencies
+pandas = pytest.importorskip("pandas")
+
+connect.import_html_libs()
+HAS_HTML_DEPS = connect._HAS_LXML or (connect._HAS_BS4 and connect._HAS_HTML5LIB)
+
+
+WRITE_FMTS = [fmt for fmt in connect.PANDAS_FMTS
+              if 'write' in connect.PANDAS_FMTS[fmt]]
+
+
+@pytest.mark.parametrize('fmt', WRITE_FMTS)
+def test_read_write_format(fmt):
+    """
+    Test round-trip through pandas write/read for supported formats.
+
+    :param fmt: format name, e.g. csv, html, json
```

Comment:
Hmm, any reason to use `epydoc` formatting in the docstring here? This does not matter because test function docstring is not exposed to users, still... why?

Reply:
That's just the PyCharm default that got in there, not really by purpose.

Reply:
maybe @cdeil as pycharm power user can chime in and tell how to change the default to be numpydoc?

Reply:
PyCharm has a docstring setting - choose "NumPy" when working on Astropy.
<img width="598" alt="Screenshot 2019-03-18 at 22 00 44" src="https://user-images.githubusercontent.com/852409/54563380-55e78080-49c9-11e9-8464-8473ef8bbc1a.png">

Reply:
Thank you!

Reply:
Cool!  My PyCharm experiment is looking better each day!